docs/tutorials/robusta-runner-metrics.rst

```diff
@@ -91,7 +99,7 @@ The response will look similar to this
       release: robusta # this label must exists on ServiceMonitors in order to be discovered.
   shards: 1
 
-* Robusta helm installation will deploy the following ServiceMonitor CRD into your cluster:
+Robusta helm installation will deploy the following ServiceMonitor CRD into your cluster:
```

Comment:
Are you saying here this will occur on it's own (when I install with helm) or are you saying I need to modify it and deploy myself? (If so, how?)

Reply:
Will updated after clarifying with @RoiGlinik. Merging for now.